sct.py

```diff
@@ -669,73 +670,73 @@ def list_images(cloud_provider: str, branch: str, version: str, region: str, arc
 
     branch = branch or "master:latest"
 
-    if version is not None:
-        match cloud_provider:
-            case "aws":
-                rows = get_ami_images_versioned(region_name=region, arch=arch, version=version)
-                click.echo(
-                    create_pretty_table(rows=rows, field_names=version_fields_with_tag_name).get_string(
-                        title=f"AWS Machine Images by Version in region {region}")
-                )
-            case "gce":
-                if arch:
-                    #  TODO: align branch and version fields once scylla-pkg#2995 is resolved
-                    click.echo("WARNING:--arch option not implemented currently for GCE machine images.")
-                rows = get_gce_images_versioned(version=version)
-
-                click.echo(
-                    create_pretty_table(rows=rows, field_names=version_fields).get_string(
-                        title="GCE Machine Images by version")
-                )
-            case "azure":
-                if arch:
-                    click.echo("WARNING:--arch option not implemented currently for Azure machine images.")
-                azure_images = azure_utils.get_released_scylla_images(scylla_version=version, region_name=region)
-                rows = []
-                for image in azure_images:
-                    rows.append(['Azure', image.name, image.unique_id, 'N/A'])
-                click.echo(
-                    create_pretty_table(rows=rows, field_names=version_fields).get_string(
-                        title="Azure Machine Images by version")
-                )
-
-            case _:
-                click.echo(f"Cloud provider {cloud_provider} is not supported")
-
-    elif branch:
-        if ":" not in branch:
-            branch += ":all"
-
-        match cloud_provider:
-            case "aws":
-                region = region or "eu-west-1"
-                ami_images = get_ami_images(branch=branch, region=region, arch=arch)
-                click.echo(
-                    create_pretty_table(rows=ami_images, field_names=branch_fields_with_tag_name).get_string(
-                        title=f"AMI Machine Images for {branch} in region {region}"
+    for region in regions:
```

Comment:
It could keep indentation as-is and be smaller change from git point of view by doing following:

```
if version is None:
    return
for region in regions:
    <all the existing code stays as-is>
```